articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,88 @@
+# :name_badge: Semantic HTML Elements
+
+For this article, I assume you have a basic working knowledge of HTML. If this is not the case, please [email me](mailto:maddie.horowitz@addepar.com) and I will bring you up to speed.
+
+### HTML5
+
+HTML5 was the newest iteration of of HTML published in 2014. This version primarily focused on enabling multiple devices and systems to interpret the markup languge. With a growing focus on responsive websites, screen readers and accessibility, the WWW introduced semantic tags. Specifically, they introduced new tags to represent new formats of media (`<canvas>`, `<video>`, `<audio>`), as well as ones to represent page structure (`<main>`, `<section>`, `<header>`, `<footer>`, `<nav>`, etc). For a full list of all available elements offered as of HTML 5, please refer to the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).
+
+### Why are semantic tags better?
+
+The goal of semantic HTML elements is to better represent the content within. A `<div>` can represent just about any block-level element, but it is better to be more specific. Screen readers, parsers, and search engines that try to read your HTML and attempt to understand its structure. If you use semantic tags. you are giving these systems an easier time.
+
+It also makes your code significantly easier to read by other humans. In a complex or large page with levels and levels of `<div>`s, it becomes impossible to understand the flow of information. Too many `<div>`s can fill up a page until it becomes unreadable. This is called _divitis_. Avoid it at all costs.
+
+However, it is much easier to understand if you take advantage of elements like `<main>`, `<section>`, `<header>`, etc. because it helps you visualize the flow of information as you read through the HTML.
+
+An additional benefit of using semantic tags, is that you can write more general CSS selectors. For a refresher on specificity of CSS selectors, read the [blog post](2016-08-02-why-you-should-never-use-important.md) I wrote on the subject). The benefit of general selectors is that you can establish a foundation of styles (for example, all `<section>`s have a blue background and 10 pixels of padding), that can easily be overwritten in a more specific situation, without having to add too many extra classes or incorporating `!important`.
```

Comment:
Hmmm, not sure how I feel about this. While it is true you can use these HTML5 tags for styling, it's commonly accepted now that styling on tag selectors is a bad practice except in rare cases like CSS resets or super general global styles. Like if you REALLY think you want every single section to have a certain style, do it at your own risk.